Create roadmap-july-2019.md

sharifsalah opened the pull request:
Add a brief blogpost to link to the SIGGRAPH page for the OpenCue BoF session.

bcipriano commented:
Great idea!

A few thoughts:

- Let's hold on merging this for now until the BoF schedule is finalized, which should be in the next day or so I believe.
- We'll also be holding a public TSC meeting on Wednesday, 31 Aug at 10AM PT. I'm asking around to see if there's a good page we can link to for more info here.
- We should also link to https://www.aswf.io/siggraph2019/ which contains info on all the different ASWF events happening Tuesday.

sharifsalah commented:
Thanks, glad you're supportive!

I've updated the post with a link to the ASWF event page, as well as a note to say we look forward to meeting attendees.

WRT to the TSC meeting, I'd recommend covering this in a follow up post. I tend to feel that posting a bit more regularly is more engaging than covering multiple events or news items in a single post, but let me know if you feel it's worth covering in this post and perhaps more detail in a follow up?

Please let me know when the BoF schedule is finalized.

Preview updated at https://5d24f7da3640ed0009ffc0fe--elated-haibt-1b47ff.netlify.com/blog/2019/07/08/opencue-roadmap/

bcipriano commented on content/en/blog/news/roadmap-july-2019.md:
```diff
@@ -0,0 +1,22 @@
+session.
+
+For more information, and to add the session to your agenda, visit the
+[event page](https://s2019.siggraph.org/presentation/?id=bof_122&sess=sess297).
```
This link is updated now with the correct time, as is the ASWF schedule that you have linked below.

sharifsalah replied:
Thanks and updated with a mention of the start time.

bcipriano commented on content/en/blog/news/roadmap-july-2019.md:
```diff
@@ -0,0 +1,22 @@
+July 30.
+
+The Academy Software Foundation (ASWF) will provide boxed lunches for this
+session.
```
This text about lunch isn't relevant anymore now that the event is at 10AM.

(It still appears on the ASWF page but I've asked them to fix this.)

sharifsalah replied:
Thanks, removed from the post.

sharifsalah commented:
I've added 'SIGGRAPH' to the page title and nav link. I've also added the ASWF image.

I had previously considered adding the promotional image, but didn't as it was text heavy and difficult to provide adequate alt text for. Instead, I've updated the post with all of the information from the image so that all users can read the information.

Direct link to updated post:

https://5d2d287452f218000d48c8bc--elated-haibt-1b47ff.netlify.com/blog/2019/07/08/opencue-roadmap-at-siggraph/

sharifsalah commented:
Also is the session going to be recorded for those that can't attend? Are we planning to share any other summary, such as a transcript or notes?

bcipriano commented:
No recording or transcript that I'm aware of. I'll be taking what notes I can though that can get tricky with an open Q&A format like we have planned, so it's likely I won't end up publishing those.

sharifsalah commented:
Thanks, no problem. I'll wait for you to approve and then publish.

bcipriano commented on content/en/blog/news/roadmap-july-2019.md:
```diff
@@ -0,0 +1,22 @@
+---
+title: "OpenCue roadmap at SIGGRAPH"
```
nit: Should "roadmap" be capitalized here? Not sure what the guidelines are on titles.

sharifsalah replied:
The style guide we're following recommends using sentence case. For future reference for anyone reviewing the history of this PR, here's the relevant guidance:

https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings
